### PR TITLE
Saving of individual position create prices

### PR DESCRIPTION
**Pull request by Klapeyron**
## Summary by CodeRabbit

- **New Features**
  - Added `createPriceLong` and `createPriceShort` fields to market positions for more detailed price tracking.

- **Bug Fixes**
  - Removed an unnecessary console log statement from the latest Oracle Price Processor for cleaner logs.

- **Enhancements**
  - Improved event processors to handle new position fields and update logic for order events.
  
- **Schema Updates**
  - Updated GraphQL schema to include `createPriceLong` and `createPriceShort` in the `Position` type.

- **Documentation**
  - Documented changes in database schema and event processing logic to reflect the new fields for market positions.

### CODE REVIEW

File: src/eventprocessor/market/orderFilledEventProcessor.ts [hunk 1]
```diff
@@ -33,6 +35,41 @@ export class OrderFilledEventProcessor implements EventProcessor {
...
+          if (offsetingPosition) {
+            const positionCreatedEvent = block.events.find(
+              (element) =>
+                (element.index === event.index - 1 ||
+                  element.index === event.index - 2 ||
+                  element.index === event.index - 3
+                  || element.index === event.index - 4) &&
+                element.callAddress?.toString() === event.callAddress?.toString() &&
+                (element.name === "Market.PositionCreated"),
+            );
```
[the remark this answers is not in the document]
Comment (Klapeyron) [reply src/eventprocessor/market/orderFilledEventProcessor.ts:54]: we need to ensure, that element is before event, not after, we are searching backwards

**Comment by Klapeyron** [comment]
@coderabbitai review

**Comment by Klapeyron** [comment]
@coderabbitai summary

**Review by TomaszGol — APPROVED** [review]

**Review by TomaszGol — DISMISSED** [review]

**Review by TomaszGol — APPROVED** [review]

File: src/eventprocessor/market/orderFilledEventProcessor.ts [hunk 2]
```diff
@@ -33,6 +34,43 @@ export class OrderFilledEventProcessor implements EventProcessor {
...
+            const positionCreatedEvent = block.events.find(
+              (element) =>
+                element.index < event.index &&
+                event.index - element.index <= 4 &&
+                element.callAddress?.toString() ===
+                  event.callAddress?.toString() &&
+                element.name === "Market.PositionCreated",
+            );
+            if (positionCreatedEvent) {
+              let newPosition = await ctx.store.findOne(Position, {
```
[the remark this answers is not in the document]
Comment (Klapeyron) [reply src/eventprocessor/market/orderFilledEventProcessor.ts:54]: it is the same, but different formatting

**Pull request merged**